src/test/java/ay2122s1_cs2103t_w16_2/btbb/model/order/OrderTest.java

```diff
@@ -1,53 +1,65 @@
 package ay2122s1_cs2103t_w16_2.btbb.model.order;
 
+import static ay2122s1_cs2103t_w16_2.btbb.logic.commands.CommandTestUtil.VALID_NAME_AMY;
 import static ay2122s1_cs2103t_w16_2.btbb.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_ALICE;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_BENSON;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_CARL;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_DANIEL;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_ELLE;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_ALICE;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_BENSON;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_CARL;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_DANIEL;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_ELLE;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import org.junit.jupiter.api.Test;
 
+import ay2122s1_cs2103t_w16_2.btbb.model.client.Address;
+import ay2122s1_cs2103t_w16_2.btbb.model.client.Name;
 import ay2122s1_cs2103t_w16_2.btbb.model.client.Phone;
 import ay2122s1_cs2103t_w16_2.btbb.testutil.OrderBuilder;
 
 class OrderTest {
     @Test
     public void isSameOrder() {
```

Comment:
Could you add a check for
`// different object, same clientName, clientPhone and clientAddress -> returns true`
This method will continuously be evolved later on